src/blog/v8-release-84.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'V8 release v8.4'
+author: 'Camillo Bruni, '
+avatars:
+ - 'camillobruni'
+date: 2020-06-30
+tags:
+ - release
+description: 'V8 v8.4'
```

Comment:
Please enhance description with tl;dr of updates / features, see https://github.com/v8/v8.dev/pull/393/files#diff-db8493c22c4eb15ac498cdbe8cc9fb9dR9 for an example.

Reply:
done.